package/MDAnalysis/core/groups.py

```diff
@@ -725,6 +726,20 @@ def dimensions(self):
     def dimensions(self, dimensions):
         self.universe.trajectory.ts.dimensions = dimensions
 
+    @staticmethod
+    def _create_unique_group(_unique):
```

Comment:
I'm a bit unhappy with the function name here as it doesn't create a group but only sets its caches.
Plus you could set the ``issorted`` property here as well, I suppose.